src/ui/components/abstract/LayoutFrame.mjs

```diff
@@ -0,0 +1,30 @@
+class LayoutFrame {
+  constructor(ui) {
+    this.ui = ui;
+
+    // TODO: Risk of flags prop clashing with other classes in hierarchy?
+    this.flags = 16;
```

Comment:
Initial value of flags for `LayoutFrame` should be `0`.

Reply:
Done! 👍